hs/t/y/Fix_Throw_No_Catch.rsh

```diff
@@ -0,0 +1,13 @@
+'reach 0.1';
+
+export const main =
+  Reach.App(
+    {},
+    [Participant('A', {})],
+    (A) => {
+      try {
```

Comment:
If we add
```
A.publish();
commit();
```
Just before the `try` then that will remove warning RW0004 and will make a cleaner `txt` output.